src/policy_evaluator_builder.rs

```diff
@@ -42,6 +42,7 @@ pub struct PolicyEvaluatorBuilder {
     wasmtime_cache: bool,
     epoch_deadlines: Option<EpochDeadlines>,
     ctx_aware_resources_allow_list: HashSet<ContextAwareResource>,
+    minimal_kubewarden_version: Option<String>,
```

Comment:
I **think** we don't need this field here. This field is needed only to perform a comparison between the kubewarden stack supported by the host and the one required by the policy.
The problem is the kubewarden stack cannot be inferred by `policy-evaluator` as a library. This is something can be done only by the host (kwctl/policy-server).

Because of that, I would try to write the check on the host side, roughly:

* pull the policy
* read the policy metadata
* perform the version check -> exit with an error
* create `PolicyEvaluatorBuilder`, life goes on as usual